main.py

```diff
@@ -163,9 +165,9 @@ def api_handler(self) -> object:
     client = MechaClient(**client_args)
     await client.connect(hostname=config['irc']['server'],
                          port=config['irc']['port'],
-                         tls=config['irc']['tls'])
+                         tls=config['irc']['tls'],
+                         )
```

Comment:
Stray comma & newline. Does Python not complain when it gets here?

Reply:
No, its legal. It also translates to less of a headache if we ever want to add an additonal kwarg here, as the comma is already in place. functionally this makes no change (I think it found its way there while debugging something)